packages/shared/src/components/cards/common/list/ActionButtons.tsx

```diff
@@ -45,11 +48,22 @@ export default function ActionButtons({
   const { onInteract, interaction, previousInteraction } = usePostActions({
     post,
   });
-  const postUiExp = useFeature(featurePostUiImprovements);
+  const buttonExp = useFeature(featureCardUiButtons);
+  const colorExp = useFeature(featurePostUiImprovements);
   const isFeedPreview = useFeedPreviewMode();
   const { data, onShowPanel, onClose } = useBlockPostPanel(post);
   const { showTagsPanel } = data;
 
+  // Experiment configuration
+  const config = {
+    useNewExperience: colorExp || buttonExp,
+    showAwardAction: buttonExp,
+    copyButtonColor: colorExp ? ButtonColor.Water : ButtonColor.Cabbage,
+    copyButtonClassName: colorExp ? 'hover:text-text-link' : undefined,
+    background: colorExp ? 'bg-transparent' : 'bg-surface-float',
+    iconSize: buttonExp ? IconSize.Medium : IconSize.Medium, // Keep Medium for list view
+  };
```

Comment:
Can we abstract this maybe in a helper function/hook?
It being re-used makes for potentially issues on cleanup/additional tests